### PR TITLE
Custom splash screen

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Fixes https://github.com/inventree/InvenTree/issues/3487

Adds the ability to specify a custom splash screen for the various login screens

e.g.

image

### CODE REVIEW

File: InvenTree/templates/account/base.html [hunk 1]
```diff
@@ -49,10 +49,8 @@
 {% endblock %}
 </head>
 
-<body class='login-screen'>
-    <!--
-        Background Image Attribution: https://unsplash.com/photos/Ixvv3YZkd7w
```
Comment (matmair) [inline InvenTree/templates/account/base.html:54]: Suggestion: Maybe we should add this attribution to the credits in the docs?

File: InvenTree/InvenTree/settings.py [hunk 2]
```diff
@@ -839,8 +839,18 @@
...
 
+# Check for a custom splash screen
+CUSTOM_SPLASH = get_setting('INVENTREE_CUSTOM_SPLASH', 'customize.splash', None)
+
+if CUSTOM_SPLASH:
+    if static_storage.exists(CUSTOM_SPLASH):
+        logger.info(f"Loading custom splash screen from static directory: {CUSTOM_SPLASH}")
+    else:
+        logger.warning(f"The custom splash screen '{CUSTOM_SPLASH}' could not be found in the static directory")
+        CUSTOM_SPLASH = False
```
Comment (matmair) [inline InvenTree/InvenTree/settings.py:850]: Suggestion: Maybe this section could be refactored together with CUSTOM_LOGO to remove duplication?

**Review by matmair — APPROVED** [review]

File: InvenTree/templates/account/base.html (hunk 1 above)
Comment (SchrodingersGat) [reply InvenTree/templates/account/base.html:54]: Yeah docs PR incoming

File: InvenTree/InvenTree/settings.py (hunk 2 above)
Comment (SchrodingersGat) [reply InvenTree/InvenTree/settings.py:850]: Could do, I'll leave that for a future PR though

**Pull request merged**